docker/build/x86_64/Dockerfile

```diff
@@ -32,7 +32,12 @@ RUN make flags-release flags-pie stash
 
 # Final Runnable Image
 FROM alpine:latest
-RUN apk add --no-cache ca-certificates vips-tools ffmpeg
+RUN apk add --no-cache --virtual .build-deps gcc python3-dev musl-dev \
```

Comment:
Python was missing from this "quick" image used locally.
Could not test scrapers without it.